lib/screens/v2/candidate_generator/dup/departures.ex

```diff
@@ -224,15 +237,18 @@ defmodule Screens.V2.CandidateGenerator.Dup.Departures do
       true ->
         # Add overnight departures to the end.
         # This allows overnight departures to appear as we start to run out of predictions to show.
-        departures = departures ++ overnight_schedules_for_section
-
-        visible_departures =
-          if is_only_section do
-            Enum.take(departures, 4)
+        departures =
+          if length(departures) < max_visible_departures and overnight_schedules_for_section != [] do
+            # Temporary logs for insight into a bug with incorrect departure for next day being shown
+            # https://app.asana.com/1/15492006741476/project/1185117109217413/task/1210559658847355
+            LogScreenData.log_dup_data(screen_name, departures, overnight_schedules_for_section)
```

Comment:
I also considered adding the raw schedule data that we are pulling from the API in `get_overnight_schedules_for_section` by adding an additional field to be returned by this function. I think that could be helpful for us to have logged, but will definitely make this messier just for the sake of debugging